docs/source/FAQ/team_management_migration.rst

```diff
@@ -0,0 +1,80 @@
+================================
+Teams User Management Migration
+================================
+
+Summary
+=======
+
+This is an FAQ about migrating your code to the :class:`~citrine.resources.team.Team` version of the Citrine Platform.
+
+What's new?
+====================
+
+Teams
+------
+
+The biggest fundamental change that affects Citrine Python when migrating to CP2 is the
```

Comment:
I think you said yesterday we're not supposed to mention CP2?

Reply:
Although given the context and intro of the page, you can probably shorten this sentence to "The biggest change is the introduction..."